<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamAnalyzeWidgetClass</class>
 <widget class="QMainWindow" name="StreamAnalyzeWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>977</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StreamAnalyzeWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,1,1,2,0,0,0,6">
      <item>
       <widget class="QPushButton" name="pushButtonPlayer">
        <property name="text">
         <string>Player</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonStop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>读取NalUnit 长度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLength"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>文件路径</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditFilePath"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>码流类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxStreamType">
        <item>
         <property name="text">
          <string>H265</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>H264</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFrame">
        <property name="text">
         <string>ShowFrame</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,3,0" columnstretch="1,3">
      <item row="0" column="0" rowspan="3">
       <widget class="QTableView" name="tableViewNALU">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>0</height>
         </size>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextBrowser" name="textBrowserHead"/>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QTreeView" name="treeViewH26X"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,2,2">
      <item>
       <widget class="QTableView" name="tableViewBit">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditBit"/>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditMessage"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1236</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionclose"/>
    <addaction name="actionsave"/>
    <addaction name="separator"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
